### ISSUE TITLE
Signed URL without any transformations or query parameters is invalid

### ISSUE THREAD

**Issue by nbolender** [imagekit-developer/imagekit-php#48]
```php
$imagekit->url(['path' => '/image.jpg', 'signed' => true]);
```

Expected output: `https://ik.imagekit.io/endpoint/image.jpg?ik-s=signature`

Actual output: `https://ik.imagekit.io/endpoint/image.jpg&ik-s=signature` (401)

Workaround: add `?` to the end of `path`

**Comment by msheheryar1** [comment]
> ```
> $imagekit->url(['path' => '/image.jpg', 'signed' => true]);
> ```
> 
> Expected output: `https://ik.imagekit.io/endpoint/image.jpg?ik-s=signature`
> 
> Actual output: `https://ik.imagekit.io/endpoint/image.jpg&ik-s=signature` (401)
> 
> Workaround: add `?` to the end of `path`

Thanks for pointing out this issue, I'm looking into this.

**Comment by imagekitio** [comment]
Fixed in v 3.0.1
